game/execname.cc

```diff
@@ -14,5 +14,7 @@
 #elif (BOOST_OS_LINUX)
 #include "platform/execname.unix.inc"
 #else
+fs::path execname() {
 	return fs::path();
-#endif // BOOST_OS_WINDOWS
\ No newline at end of file
+}
+#endif // BOOST_OS_WINDOWS
```

Comment:
do you prefer an
```
#error OS NOT SUPPORTED, PLEASE DEFINE FS::PATH
``` line?

The issue I got was by chance TBH
for some reasons on *linux* gcc doesn't set on ppc64el architecture the __linux macro when std=c++14 is set, this leads to boost not defining linux system, and the build to fail.

After debugging we got a patch against boost, that will fix the issue
https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=914688
https://buildd.debian.org/status/fetch.php?pkg=performous&arch=ppc64el&ver=1.1%2Bgit20181118-2&stamp=1543243322&raw=0

Probably going with an #error string is the safest assumption to do (since the original bug was actually in boost code)

this is the boost fix:
https://github.com/boostorg/predef/commit/d58fcca2d59faae030d3cbeb68da6a7c63c906d7
you should probably cherry-pick into your local fork version

Reply:
@LocutusOfBorg alternatively to what I said earlier... is the issue fixed if we use `BOOST_OS_UNIX` instead of `BOOST_OS_LINUX` ?